README.rst

```diff
@@ -78,8 +78,7 @@ Hipchat tab. Enter the required credentials and click save changes.
 Slack
 -------
 
-Go to your project's configuration page (Projects -> [Project]) and select the
-Slack tab. Enter the required credentials and click save changes.
+As of March 28, 2018, Slack is part of core Sentry and no longer a plug-in. To configure Slack with your on-prem Sentry please see [this forum post](https://forum.sentry.io/t/how-to-configure-slack-in-your-on-prem-sentry/3463/2).
```

Comment:
You might want to link just to `https://forum.sentry.io/t/how-to-configure-slack-in-your-on-prem-sentry/3463`

The url with `/2` ends up being a deep link to the bottom of the page. :)

Reply:
whoops! removed. ty @mattrobenolt